symbolic/am_and.m

```diff
@@ -1,6 +1,11 @@
 function fun = am_and(a,b)
-% syms x y
-% f = symfun(sym('cw_and(x,y)'),[x y]);
-% fun = f(a,b);
-fun = a*b;
+% am_and is the amici implementation of the symbolic and function
+%
+% Parameters:
+%  a: first input parameter @type sym
+%  b: second input parameter @type sym
+%
+% Return values:
+%  fun: logical value, negative for false, positive for true
+fun = am_min(a,b);
```

Comment:
with the mapping

true -> positive
false -> negative

minimum  is only positive (true) if both a and b are positive (true)